### DISCUSSION TITLE
ResourceLoader.GetForViewIndependentUse().GetString() is not worked

### DISCUSSION THREAD

**Discussion by AJ-comp** [unoplatform/uno#15934] [Q&A]
I'm new to the Uno platform, but I'm familiar with the mvvm pattern through wpf.

The question I want to ask today is about Uno's multilingual system, unrelated to the MVVM pattern.

I am using Uno's default template, and there are resw files as shown below.

![image](https://github.com/unoplatform/uno/assets/69152847/ed514261-b993-4014-93c5-f7973a9091f0)

Every resw file has a key called "Loading" and a corresponding string value.

![image](https://github.com/unoplatform/uno/assets/69152847/518cc3ec-5783-44ec-84a2-0bb7416116ab)

And I created a structure so that the RegisterCommandAsync function is called when the Register button is clicked.

```
    public async Task RegisterCommandAsync()
    {
        if (Nickname == string.Empty)
        {
            var pp = Windows.ApplicationModel.Resources.ResourceLoader
                    .GetForViewIndependentUse()
                    .GetString("Loading");

            NickNameError = "Input Nickname";
            return;
        }
    }
```

  Inside the RegisterCommandAsync function, the ErrorMessage is written to retrieve the string value for the Loading key through the multilingual code recommended by the Uno Platform, as shown in the code below.

var pp = ResourceLoader.GetForViewIndependentUse().GetString("Loading");

However, even though there is a Loading key value, the value of pp is empty.
Can anyone tell me what I missed? Any help would be appreciated.

**Comment by jeromelaban** [comment]
Thanks for the report, see https://github.com/unoplatform/uno/discussions/15220#discussioncomment-8335334 for a  resolution.
